### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by 300ms**
Project has been created and gems are installed. This pull request has been created to get a code review.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+group :test do
+  # Adds support for Capybara system testing and selenium driver
+  gem 'capybara', '>= 2.15'
+  gem 'selenium-webdriver'
+  # Easy installation and use of web drivers to run system tests with browsers
+  gem 'guard'
+  gem 'guard-minitest'
+  gem 'minitest'
+  gem 'minitest-reporters'
+  gem 'pry'
```
Comment (samgaco) [inline Gemfile]: - [x] you should rather use rspec for the testing since that is what is taught in microverse.

you can add it with `gem 'rspec-rails'`

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @WCanirinka and @300ms :high_brightness: 

- [x] You should have [Devise](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone2/device.png)  and [Omniauth](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone2/omniauth.png) set up in your gem file for this milestone.


Please update those and submit back for another review.
-------------------------------
* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Comment by 300ms** [comment]
Removed the 'minitest', 'minitest-reporters' and 'pry' gems. Installed rspec instead of them to use in our tests. Also Devise and Omniauth gems are installed.

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: :clap: 

You can merge it and move on :+1: 
Good luck with the next milestone :smile: 

Thanks
**Mael FOSSO**
[Github](https://github.com) | [Twitter](https://twitter.com) | [LinkedIn](https://www.linkedin.com/in/mael-elvis-fosso-650b6346/)

**Pull request merged**